packages/nimble-components/src/wafer-map/modules/matrix-renderer.ts

```diff
@@ -0,0 +1,17 @@
+import { wrap, Remote } from 'comlink';
+import type { WaferMap } from '..';
+import { workerCode } from '../workers/render-worker';
+import type { RenderWorker } from '../../../build/generate-workers/dist/esm/source/render-worker';
+
+/**
+ * Responsible for drawing the dies inside the wafer map, adding dieText and scaling the canvas
+ */
+export class MatrixRenderer {
+    public readonly workerOne: Remote<RenderWorker>;
+
+    public constructor(private readonly wafermap: WaferMap) {
+        const blob = new Blob([workerCode], { type: 'text/javascript' });
+        const url = URL.createObjectURL(blob);
```

Comment:
Need to make sure to clean-up the objectUrl after the worker is made.
Either that that or lets just make it once lazily instead of every construction. (that might be better)